packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -313,20 +322,28 @@ export abstract class ApplicationLoadBalancedServiceBase extends cdk.Construct {
     if (props.certificate !== undefined && props.protocol !== undefined && props.protocol !== ApplicationProtocol.HTTPS) {
       throw new Error('The HTTPS protocol must be used when a certificate is given');
     }
+    let containerPort = 80;
+    if (props.taskImageOptions !== undefined && props.taskImageOptions.containerPort !== undefined) {
```

Comment:
This only allows providing a custom container port when using `taskImageOptions`, but it does not allow specifying a custom port when using a task definition. Exposing the `containerPort` as optional (defaulting to 80 if unspecified) would make that more accessible.